sklearn/linear_model/tests/test_logistic.py

```diff
@@ -32,6 +32,10 @@
     LogisticRegressionCV,
 )
 
+pytestmark = pytest.mark.filterwarnings(
+    "error::sklearn.exceptions.ConvergenceWarning:sklearn.*"
+)
+
```

Comment:
We can maybe be a bit more lenient now :)
I would like this thing to be at the level of the package at some point.